webrisk/snippets/compute_threatlist_diff.py

```diff
@@ -0,0 +1,81 @@
+#  limitations under the License.
+
+# [START webrisk_compute_threatlist_diff]
+from google.cloud import webrisk_v1
+
+
+def compute_threatlist_diff(threat_type: webrisk_v1.ThreatType, version_token: bytes, max_diff_entries: int,
+                            max_database_entries: int, compression_type: webrisk_v1.CompressionType) -> None:
+    """
+    Gets the most recent threat list diffs. These diffs should be applied to a local database of
+    hashes to keep it up-to-date.
+
+    If the local database is empty or excessively out-of-date,
+    a complete snapshot of the database will be returned. This Method only updates a
+    single ThreatList at a time. To update multiple ThreatList databases, this method needs to be
+    called once for each list.
+
+    Args:
+        threat_type: The threat list to update. Only a single ThreatType should be specified per request.
+                     threat_type = webrisk_v1.ThreatType.MALWARE
+
+        version_token: The current version token of the client for the requested list. If the client does not have
+                       a version token (this is the first time calling ComputeThreatListDiff), this may be
+                       left empty and a full database snapshot will be returned.
+
+        max_diff_entries: The maximum size in number of entries. The diff will not contain more entries
+                          than this value. This should be a power of 2 between 2**10 and 2**20.
+                          If zero, no diff size limit is set.
+                          max_diff_entries = 1024
+
+        max_database_entries: Sets the maximum number of entries that the client is willing to have in the local database.
+                              This should be a power of 2 between 2**10 and 2**20. If zero, no database size limit is set.
+                              max_database_entries = 1024
+
+        compression_type: The compression type supported by the client.
+                          compression_type = webrisk_v1.CompressionType.RAW
+    """
+
+    webrisk_client = webrisk_v1.WebRiskServiceClient()
+
+    constraints = webrisk_v1.ComputeThreatListDiffRequest.Constraints()
+    constraints.max_diff_entries = max_diff_entries
+    constraints.max_database_entries = max_database_entries
+    constraints.supported_compressions = [compression_type]
+
+    request = webrisk_v1.ComputeThreatListDiffRequest()
+    request.threat_type = threat_type
+    request.version_token = version_token
+    request.constraints = constraints
+
+    response = webrisk_client.compute_threat_list_diff(request)
+
+    # The returned response contains the following information:
+    # https://cloud.google.com/web-risk/docs/reference/rpc/google.cloud.webrisk.v1#computethreatlistdiffresponse
+    # Type of response: DIFF/ RESET/ RESPONSE_TYPE_UNSPECIFIED
+    print(response.response_type)
+
+    # List of entries to add and/or remove.
+    # print(response.additions)
+    # print(response.removals)
```

Comment:
Should these be commented out?

Reply:
Yep, we wouldn't want to print out the hashes. 
Hence, commented it.